runelite-client/src/main/java/net/runelite/client/plugins/virtuallevels/VirtualLevelsConfig.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2018, Joshua Filby <joshua@filby.me>
```

Comment:
Neither of us wrote this file, you should use your own copyright attribution for this.